ci/infra/aws/security-groups.tf

```diff
@@ -102,96 +113,120 @@ resource "aws_security_group" "allow_control_plane_traffic" {
     "Name", "${var.stack_name}-control-plane",
     "Class", "SecurityGroup"))}"
 
+  # etcd - internal
   ingress {
-    from_port   = 2380
+    from_port   = 2379
     to_port     = 2380
     protocol    = "tcp"
     cidr_blocks = ["${var.vpc_cidr_block}"]
   }
 
+  # VXLAN traffic (cilium) - internal
   ingress {
-    from_port   = 8285
-    to_port     = 8285
+    from_port   = 8471
```

Comment:
According to https://docs.cilium.io/en/v1.4/install/system_requirements/#firewall-rules, `8472` should suffice. Am I missing something? [1]

Reply:
Yeah, this is odd. @mrostecki should we drop port `8471`?

Reply:
Dropped